HISTORY.md

```diff
@@ -17,6 +17,7 @@
 * Disable file deletion after MANIFEST write/sync failure until db re-open or Resume() so that subsequent re-open will not see MANIFEST referencing deleted SSTs.
 * Fix a bug when index_type == kTwoLevelIndexSearch in PartitionedIndexBuilder to update FlushPolicy to point to internal key partitioner when it changes from user-key mode to internal-key mode in index partition.
 * Make compaction report InternalKey corruption while iterating over the input.
+* Fix a bug in MultiRead, which may cause it to read more data than needed, but won't affect correctness.
```

Comment:
MultiRead() is an internal interface. Can you mention the end impact to user? I guess it's to cause MultiGet() to be slower? Also it's might be a good idea to mention which release it got impacted.